lib/index.js

```diff
@@ -54,9 +50,11 @@ function onPostAuthFactory(qsOptions) {
         request.payload = qs.parse(request.payload, qsOptions);
     }
 
-    return reply.continue();
+    return h.continue;
   };
 };
 
+register.attributes = { pkg: require('../package.json') };
+exports.name = require('../package.json')['name'];
```

Comment:
Are you sure this works? From hapi release notes https://github.com/hapijs/hapi/issues/3658 (Plugins section, sorry I cannot link it directly) 
```
Replace the exports.register() and the matching exports.register.attributes with exports.plugin = { register, name, version, multiple, dependencies, once, pkg }
```

looks like the expected object is 
```
exports.plugin = { 
  register, 
  pkg: require('../package.json')
}
```

Reply:
I will verify again -- but we are using this in our project :) The Hapi docs on the migration are not perfect is what I discovered.

Reply:
So it looks like the new syntax works as well, so I moved towards that, thanks for noticing :) Also looks like the plugin docs are still out of date.